docs/development/scripts/install-kubevirt.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+KUBEVIRT_VERSION="v0.29.0"
+CDI_VERSION="v1.18.0"
```

Comment:
These variables can be put in a `common.sh` file and sourced in the other scripts instead of defining them on multiple places.

Reply:
Done